src/ODbL-1.0.xml

```diff
@@ -7,11 +7,11 @@
       </crossRefs>
     <text>
       <titleText>
-	     <optional>##</optional>
+        <alt match="header1" match="#*"/>
```

Comment:
Probably intended name="header1".

Two elements called match is illegal XML, causing travis build fail:
https://travis-ci.org/spdx/license-list-XML/builds/398415897

Reply:
> Probably intended name="header1".

Oops, thanks.  Fixed with 2729eed -> 23d2671.